maintainers/maintainer-list.nix

```
@@ -7318,6 +7318,12 @@
     githubId = 8547242;
     name = "Stefan Rohrbacher";
   };
+  "thelegy" = {
+    email = "mail+nixos@0jb.de";
+    github = "thelegy";
+    githubId = "3105057";
+    name = "Jan Beinke";
+  };
```

this should be a separate commit
```
maintainers: add thelegy
```
```suggestion
  "thelegy" = {
    email = "mail+nixos@0jb.de";
    github = "thelegy";
    githubId = 3105057;
    name = "Jan Beinke";
  };
```

---

i splitted the commits and force pushed to this pr